tools/esp8266/app.cpp

```diff
@@ -566,6 +566,11 @@ void app::cbMqtt(char* topic, byte* payload, unsigned int length) {
                                 // uint16_t power_factor = std::stoi(strtok(NULL, "/"));
                                 DPRINTLN(DBG_INFO, F("Set Power Factor not implemented for inverter ") + String(iv->id) );
                                 break;
+                            case CleanState_LockAndAlarm: // Clean lock & alarm (+ reboot)
```

Comment:
Reboot should not be necessary / forced.
If this is a malfunction, we need to sort it out.
Should we add the other DevInfo SubCmd Types just the same way using several case's ?